browser/resources/brave_new_tab_page_refresh/components/common/popover.tsx

```diff
@@ -15,7 +15,7 @@ interface Props {
 export function Popover(props: Props) {
   const elementRef = React.useRef<HTMLDivElement>(null)
 
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
This change is to make sure these popover elements are set up correctly before the first paint. This component will soon go away in favor of `Floating`, etc. - this is an interim improvement.